framework/wazuh/core/cluster/worker.py

```diff
@@ -719,36 +727,42 @@ def overwrite_or_create_files(filename_: str, data_: Dict):
                           )
 
         errors = {'shared': 0, 'missing': 0, 'extra': 0}
+        result_logs = {'debug': defaultdict(list), 'debug2': defaultdict(list),
+                       'error': defaultdict(list), 'generic_errors': []}
 
         for filetype, files in ko_files.items():
             # Overwrite local files marked as shared or missing.
             if filetype == 'shared' or filetype == 'missing':
-                logger.debug(f"Received {len(ko_files[filetype])} {filetype} files to update from master.")
+                result_logs['debug'][filetype].append(f"Received {len(ko_files[filetype])} {filetype} files to "
```

Comment:
I would remove this log since the same information is already printed here:
https://github.com/wazuh/wazuh/blob/0eeb08d9a3c3ce8bfbb941c77bc5bddb2ae85d76/framework/wazuh/core/cluster/worker.py#L646-L647